### ISSUE TITLE
Update the GitHub action to push to Quay.io

### ISSUE THREAD

**Issue by christophe-f** [janus-idp/backstage-showcase#81]
Once the permission to Quay is resolved, we need to do the following tasks:

- [ ] Update the GitHub Action to push the image to quay.io instead of GitHub registry.
- [ ] Use the image from Quay to deploy the showcase app in the ArgoCD manifest.
- [ ] Update the annotation in the `catalog-info.yaml` file.

**Comment by SamoKopecky** [comment]
I can take care of this, feel free to assign me.

**Comment by tumido** [comment]
I've updated the Janus IDP organization config to provide `QUAY_USERNAME` variable and `QUAY_TOKEN` secret at organization level - this way the credentials previously available only to the `redhat-backstage-build` repo are now available to every public repository in this organization.

I've also created the https://quay.io/repository/janus-idp/backstage-showcase repository and added write permissions for the `janus-idp+backstage_pusher` robot account.

Additionally @christophe-f you've been added as the quay org admin.
